keys/keys.go

```diff
@@ -0,0 +1,55 @@
+package keys
```

Comment:
This seems like it's from a separate PR. Isn't being used anywhere, and is unrelated to the rest of the change. Could you stash and save it for later?

Reply:
whoops, yes this shouldn't be here